rest_framework_json_api/parsers.py

```diff
@@ -4,6 +4,8 @@
 from rest_framework import parsers
 from rest_framework.exceptions import ParseError
 
+from django.conf import settings
```

Comment:
I've actually copied the style applied here from utils.py file:
https://github.com/django-json-api/django-rest-framework-json-api/blob/develop/rest_framework_json_api/utils.py